CHANGELOG.md

```diff
@@ -97,20 +99,87 @@
+  This change effectively changes the inlined namespace from `gcpcxxV1` to `v1`.
+
+**OTHER CHANGES**
+* feat: create relocatable pkg-config files ([#7481](https://github.com/googleapis/google-cloud-cpp/pull/7481))
+
+### [Bigtable](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/bigtable/README.md)
+
+* doc: remove inline namespace from doxygen ([#7461](https://github.com/googleapis/google-cloud-cpp/pull/7461))
+* fix(bigtable): retry internal errors known to be transient ([#7395](https://github.com/googleapis/google-cloud-cpp/pull/7395))
+
 ### Firestore
 
 **BREAKING CHANGE**: The _experimental_ Firestore support library that used to
 live in this repo at `google/cloud/firestore` has been removed in favor of the
 canonical library at https://firebase.google.com/docs/reference/cpp. For more
 info see [#7443](https://github.com/googleapis/google-cloud-cpp/issues/7443).
 
+* cleanup(firestore)!: removes the experimental firestore library ([#7468](https://github.com/googleapis/google-cloud-cpp/pull/7468))
+
+### [IAM](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/iam/README.md)
+
+**BREAKING CHANGE**
+* refactor(generator)!: use GOOGLE_CLOUD_CPP_NS for inline namespace ([#7439](https://github.com/googleapis/google-cloud-cpp/pull/7439))
+  This change effectively changes the inlined namespace from `gcpcxxV1` to `v1`.
+
+**OTHER CHANGES**
+* feat: create relocatable pkg-config files ([#7481](https://github.com/googleapis/google-cloud-cpp/pull/7481))
+* fix(generator): add doxygen return comment for StreamRanges ([#7419](https://github.com/googleapis/google-cloud-cpp/pull/7419))
+
+### [Pub/Sub](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/pubsub/README.md)
+
+* feat: create relocatable pkg-config files ([#7481](https://github.com/googleapis/google-cloud-cpp/pull/7481))
+* doc: remove inline namespace from doxygen ([#7461](https://github.com/googleapis/google-cloud-cpp/pull/7461))
+* feat(pubsub): implement GUAC for Subscriber ([#7449](https://github.com/googleapis/google-cloud-cpp/pull/7449))
+* feat(common): GUAC for async read-write streams ([#7442](https://github.com/googleapis/google-cloud-cpp/pull/7442))
+* feat(pubsub): implement GUAC for Publisher ([#7440](https://github.com/googleapis/google-cloud-cpp/pull/7440))
+* feat(pubsub): implement GUAC for SchemaAdmin ([#7436](https://github.com/googleapis/google-cloud-cpp/pull/7436))
+* feat(pubsub): implement GUAC for SubscriptionAdmin ([#7432](https://github.com/googleapis/google-cloud-cpp/pull/7432))
+* feat(pubsub): implement GUAC for TopicAdmin ([#7428](https://github.com/googleapis/google-cloud-cpp/pull/7428))
+* fix(pubsub): do not overwrite ConnectionOptions ([#7406](https://github.com/googleapis/google-cloud-cpp/pull/7406))
+
+### [Spanner](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/spanner/README.md)
+
+**BREAKING CHANGE**
```

Comment:
Ditto, I think we should not mark these changes as breaking because they are not (yet) breaking.